app/src/main/java/fr/free/nrw/commons/category/CategoryImagesListFragment.java

```diff
@@ -51,7 +52,7 @@
     @BindView(R.id.categoryImagesList) GridView gridView;
     @BindView(R.id.parentLayout) RelativeLayout parentLayout;
     private boolean hasMoreImages = true;
-    private boolean isLoading;
+    private boolean isLoading=true;
```

Comment:
Tiny note, but how about putting spaces around the equal sign, like above and below? That is the usual code convention as seen at https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141270.html#18761